src/web/components/App.tsx

```diff
@@ -330,6 +331,21 @@ export const App = ({ CAPI, NAV }: Props) => {
             };
             injectPrivacySettingsLink(); // manually updates the footer DOM because it's not hydrated
 
+            // keep this in sync with static/src/javascripts/boot.js in frontend
+            let recordedConsentTime = false;
+            cmp.willShowPrivacyMessage().then((willShow) => {
+                onConsentChange(() => {
+                    if (!recordedConsentTime) {
+                        trackPerformance(
```

Comment:
we already have a performance measurement lib for components `/src/web/browser/initPerf.ts`

Reply:
does it send timings to google analytics?

Reply:
no it doesn't 🙂  plus I noticed subtle differences, so all is good 👍

Reply:
cheers :) 👍